test/message-based/PriceAware.test.ts

```diff
@@ -1,4 +1,4 @@
-import { ethers } from "hardhat";
+import {ethers, waffle} from "hardhat";
```

Comment:
It'll be great to refactor the test a little bit and extract tests for different versions:
inline, editable, upgradable + a separate test with benchmarks